pyproject.toml

```diff
@@ -59,6 +60,7 @@ build-backend = "poetry.core.masonry.api"
 [tool.ruff]
 line-length = 120
 exclude = ["nominal/_api/*"]
+include = ["nominal/**/*.py", "tests/**/*.py"]
```

Comment:
is the wildcard in the path here going to match any level of subdir? is it sufficient to use something like `nominal/*`?

Reply:
no we need the .py, otherwise ruff tries to check non py files which we don't want to do right now

Reply:
<img width="723" alt="image" src="https://github.com/user-attachments/assets/055bec60-445b-4cb6-9592-1ea7c14d84db">